flink-runtime-web/web-dashboard/app/partials/jobs/job.jade

```diff
@@ -50,19 +50,15 @@ nav.navbar.navbar-default.navbar-fixed-top.navbar-main(ng-if="job")
 nav.navbar.navbar-default.navbar-fixed-top.navbar-main-additional(ng-if="job")
   ul.nav.nav-tabs
     li(ui-sref-active='active')
-      a(ui-sref=".plan") Plan
+      a(ui-sref=".plan") Overview
 
-    //- li(ui-sref-active='active' ng-if="job['end-time'] > -1")
     li(ui-sref-active='active')
       a(ui-sref=".timeline") Timeline
 
     li(ui-sref-active='active')
       a(ui-sref=".exceptions") Exceptions
 
     li(ui-sref-active='active')
-      a(ui-sref=".properties") Properties
-
-    li(ui-sref-active='active')
       a(ui-sref=".config") Configuration
```

Comment:
+1 to remove it since it doesn't help with streaming jobs. For batch it might be interesting. Did we have consensus to remove it? It's OK from my side.